### DISCUSSION TITLE
Changing the number of slots for Ender Chest.

### DISCUSSION THREAD

**Discussion by Roper89** [Zrips/CMI#2704] [Feature Requests]
Add the ability to set the number of slots for a player in his Ender Chest.

**Comment by mrfloris** [comment]
Doesn't the game define how big an enderchest is? I don't know if there will be consequences if you expand this to say 54 and then upgrade to 1.16 and things change or other unexpected situations.

**Comment by Va1a** [comment]
There are 3rd party plugins which allow echest enhancements such as this, but be warned they do not use the vanilla echest. Instead, they use a "playervault" type system. It won't be the same as the vanilla enderchest.
